src/pages/contact/index.jsx

```diff
@@ -22,21 +24,30 @@ const Contact = () => {
     }));
   };
 
-  const handleSubmit = async (e) => {
+  const form = useRef();
+
+  const sendEmail = (e) => {
     e.preventDefault();
-    setIsSubmitting(true);
 
-    await new Promise((resolve) => setTimeout(resolve, 1500));
-    setFormData({
-      name: "",
-      email: "",
-      mobile: "",
-      requirements: "",
-      city: "",
-    });
-    setIsSubmitting(false);
-    toast.success("We will get back to you soon!");
+    emailjs
+      .sendForm(
+        import.meta.env.VITE_SERVICE_ID,
+        import.meta.env.VITE_TEMPLATE_ID,
+        form.current,
+        {
+          publicKey: import.meta.env.VITE_PUBLIC_KEY,
+        }
+      )
+      .then(
+        () => {
+          toast.success("We will get back to you soon!");
+        },
+        (error) => {
+          toast.error("Unable to send your message",error.text);
```

Comment:
Yaha pura error ko log karke fir se push karo

Reply:
koi error nhi aaraha abhi ham check kiye

Reply:
> koi error nhi aaraha abhi ham check kiye

Are dev env me nahi aaraha hoga aur prod me pata kaise chalega jab error log hi nahi ho raha to?

Reply:
done